src/cameras/capture/opencv_camera/opencv_camera.py

```diff
@@ -23,11 +26,26 @@ class WebcamConfig(BaseModel):
     resolution_width: int = 800
     resolution_height: int = 600
     save_video: bool = True
-    fourcc: str = "MP4V"
-    # fourcc: str = "MJPG"
+    # fourcc: str = "MP4V"
+    fourcc: str = "MJPG"
     base_save_video_dir = _get_home_dir()
 
 
+@dataclass
+class CameraCalibrationData:
+    image_width: int
+    image_height: int
+    lens_distortion_calibration_data: LensDistortionCalibrationData = None
+    camera_translation_relative_to_camera0: np.ndarray = np.zeros((3, 1))
```

Comment:
what is "camera0"? how do we identify it?

Reply:
(btw, this bit is roughly speaking the boundary between "pure" computer vision and robotics/"perception engineering")

Reply:
This convention also has the nice feature of playing nicely with "single camera" setups - If there is only one camera attached to the computer, the world coordinates are defined to be the same as the camera coordinates things should go smoothly enough from there

Reply:
Once we're getting estimates of  3d skeletons, we can use those to estimate the *actual* world by making some pretty squishy assumptions to figure out the location and orientation of the groundplane, so that we can align things with gravity/'the inertial reference frame'